puppet-squid/src/main/java/com/iadams/sonarqube/puppet/api/PuppetGrammar.java

```diff
@@ -198,15 +193,7 @@ public static void grammar(LexerfulGrammarBuilder b) {
 
         b.rule(PARAMS).is(b.optional(
                 PARAM,
-                b.zeroOrMore(COMMA, PARAM))).skip();
```

Comment:
@racodond by removing the ADD_PARAM, you break the support for the appending to attributes: http://docs.puppetlabs.com/puppet/3.8/reference/lang_classes.html#appending-to-resource-attributes we can fix this with:
```
b.rule(PARAM).is(
  PARAM_NAME,
  b.firstOf(FARROW, PARROW),
  EXPRESSION);
```